lean-circuit/LeanCircuit/SemanticEquivalence.lean

```diff
@@ -46,6 +49,17 @@ lemma embed_dir_vector_is_binary {depth} (ix : Vector Dir depth) :
     { simp [embed_dir, is_bit]; cases d <;> simp }
     { assumption }
 
+-- End of misc lemmas
```

Comment:
there's a much better format for Lean docs, it also supports `code` etc. Please see at how mathlib is being doicumented